extensions/index.md

```diff
@@ -405,12 +405,13 @@ Property     | Type        | Value Description
 **assessmentDescription**     | string | Description of the single assessment. (required)
```

Comment:
It might be clearer to have a separate table for the AssessmentObject class, sort of following from how the core spec index.md is laid out. Perhaps it's unclear to users from this page what properties make up the AssessmentObject and what properties are in scope of the extensions:Assessment class directly.